addons/website_sale/static/src/js/website_sale_delivery.js

```diff
@@ -28,6 +28,7 @@ publicWidget.registry.websiteSaleDelivery = publicWidget.Widget.extend({
     start: async function () {
         this.carriers = Array.from(document.querySelectorAll('input[name="delivery_type"]'));
         this.keepLast = new KeepLast();
+        await this._getCurrentLocation();
```

Comment:
Could you please move it to L40 or L41? As if there are no carriers it is useless to fetch location.